addons/sale/models/product_product.py

```diff
@@ -92,3 +92,16 @@ class ProductAttributeCustomValue(models.Model):
     attribute_value_id = fields.Many2one('product.attribute.value', string='Attribute Value')
     sale_order_line_id = fields.Many2one('sale.order.line', string='Sale order line')
     custom_value = fields.Char('Custom value')
+
+
+class PricelistItem(models.Model):
+    _inherit = "product.pricelist.item"
+
+    def write(self, vals):
+        res = super(PricelistItem, self).write(vals)
+        product = self.env['product.product'].search([('product_tmpl_id','=', self.product_tmpl_id.id)])
```

Comment:
Maybe we will need iterate loop to prevent expected singleton (Incase write is called in batch)